optaplanner-website-root/content/blog/2023/04/05/BoostVRPListSelectorScalability.adoc

```diff
@@ -0,0 +1,76 @@
+https://github.com/kiegroup/optaplanner/blob/main/optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingBenchmarkConfigListNearby.xml[optaplanner-benchmark]
+ results for different amount of samples. You can find out more about optaplanner-benchmarks https://www.optaplanner.org/docs/optaplanner/latest/benchmarking-and-tweaking/benchmarking-and-tweaking.html[here].
+
+== Execution time
+
+To analyze how much nearby reduces the
+time for finding the best solution lets compare the computational speed of the Late Acceptance (LA)
+algorithm with the Nearby algorithm, and analyze how quickly each algorithm
+improves the solution score. The results indicate that the Nearby algorithm
+consistently outperforms the LA algorithm in terms of speed of improvement.
+In particular, you can see that the LV Nearby 20 variant was the fastest on average, while the Late Acceptance algorithm was the slowest.
+
+image::scoreCalculationSpeedPerSecondNearby.png[How NearByMoveSelector improves score calculation speed]
+
+[#table1]
+.Score calculation count per second comparing Nearby and Default solvers
+|===
+| ^.^|Belgium-n50-k10
+(2,890.00 samples)|Belgium-n100-k10 (10,790.00 samples)|Belgium-n500-k20
+(258,980.00 samples)|Belgium-n1000-k20 (1,017,980.00 samples)
+|Late Acceptance                      >|35,393.00/s >|37,334.00/s >|31,667.00/s >|30,282.00/s
+|LV Nearby Parabolic Distribution 10  >|40,428.00/s >|45,059.00/s >|32,708.00/s >|32,036.00/s
+|LV Nearby Parabolic Distribution 20  >|38,567.00/s >|40,552.00/s >|34,418.00/s >|37,188.00/s
+|LV Nearby Parabolic Distribution 40  >|37,551.00/s >|39,833.00/s >|33,558.00/s >|35,125.00/s
+|LV Nearby Parabolic Distribution 80  >|37,428.00/s >|35,625.00/s >|30,523.00/s >|36,060.00/s
+|LV Nearby Parabolic Distribution 160 >|41,197.00/s >|38,086.00/s >|33,724.00/s >|34,451.00/s
+|===
+
+== Solution quality
+
+Despite the fact that Nearby improves the speed of calculation, it could potentially decrease the quality of the best solution.
+
+image::bestScoreNB.png[How NearByMoveSelector affect scores]
+
+Here we see the score values for different solvers and datasets. You may see how much the nearby improves the score calculation while the dataset increases. From the other side look at the LV Nearby 10 bars. Despite being the fastest solution of all with the small dataset, it dramatically fell into local optimum and stopped improving the solution.
+
+image::belgium-n500-k20BestScoreStatisticLevel.png[How local optimum affects the score]
+
+== Scalability
+
+image::bestScoreScalabilitySummaryLevel1.png[Scalability of Nearby solver]
+
+For small datasets if the account of customers is no more than
+500 it does not matter what to choose, but starting from the 250 000 samples
+or 500 customers and 20 vehicles we can notice that the difference in the best
+scores achieved is almost 1.7X and the ratio increases to 2.63X reaching the 1m samples.
```

Comment:
Is "1m" 1 million? Let's use the full word, please, abbreviations are often confusing.